### DISCUSSION TITLE
What can I do?

### DISCUSSION THREAD

**Discussion by conversun** [swellstores/community#408] [Help] [answered]
<img width="900" alt="image" src="https://github.com/swellstores/community/assets/22893221/705f5ef6-6c8d-4f26-8fe1-eefac7bc7669">

**Comment by sags95** [comment]
@steven-k1 Should be able to help with this

**Comment by steven-k1** [comment, marked as answer]
@conversun I was able to fix the issue on our end, and you should be all good to go.
